sli/slistartup.h

```diff
@@ -40,6 +40,15 @@
 #include "slimodule.h"
 #include "token.h"
 
+// Exit codes
+#define NEST_EXITCODE_SKIPPED 200
+#define NEST_EXITCODE_SKIPPED_NO_MPI 201
+#define NEST_EXITCODE_SKIPPED_HAVE_MPI 202
+#define NEST_EXITCODE_SKIPPED_NO_OPENMP 203
+#define NEST_EXITCODE_SKIPPED_NO_GSL 204
+#define NEST_EXITCODE_SKIPPED_NO_MUSIC 205
```

Comment:
Add the other numeric values from `slistartup.cpp` here (in increasing order). These exit codes are not related to NEST as such, so drop `NEST_` from the names. Maybe add a comment that the exit code range 200-215 is reserved for test skipping exits, and point out that any new skipping codes must be added to `testsuite/do_tests.sh.in`.